[K9VULN-7866] Document limitation of global configuration (#31272)
DataDog/documentation: content/en/security/code_security/static_analysis/setup/_index.md

* Repo Level Configuration (Datadog)
* Repo Level Configuration (Repo File)

> <div class="alert alert-warning">
> By default, with no configuration defined (either at org or repo level), Datadog uses a default configuration with all default rules enabled. If you define an org-level configuration without default rules, default rules will not be used and you have to explicit enable them.

Reviewer: ```suggestion
<div class="alert alert-warning">
By default, when no configuration is defined at the org or repo level, Datadog uses a default configuration with all default rules enabled. If you define an org-level configuration without default rules, default rules are not used. If want to use default rules in this scenario, you have to enable them.
```
